webcompat/templates/home-page/form.html

```diff
@@ -29,44 +45,55 @@ <h3 class="wc-Title--s">Report Site Issue</h3>
                 {% endfor %}
             </fieldset>
           </div>
-        </div>
-        <div class="r-Grid-cell r-minM--1of2">
-          <div class="wc-Form-group wc-Form-group--biggerOffset js-Form-group {% if form.url.errors %}is-error{% endif %}">
+
+          <div class="wc-Form-group js-Form-group">
             <div class="wc-Form-information">
-              {{ form.url.label(class_='wc-Form-label') }}
-              {% if form.url.errors %}
-                {% for error in form.url.errors %}
+              {{ form.description.label(class_='wc-Form-label') }}
+              {% if form.description.errors %}
+                {% for error in form.description.errors %}
                   <span class="wc-Form-helpMessage">{{ error }}</span>
                 {% endfor %}
               {% endif %}
             </div>
             <span class="wc-Form-validation">
-            {{ form.url(class_='wc-Form-item required',
-               placeholder='e.g., http://www.example.com',
-               required=True, type='url', pattern='') }}
+            {{ form.description(class_='wc-Form-item required',
+               placeholder='e.g. The video should have started after 3 sec.',
+               required=True, type='text', pattern='') }}
             </span>
           </div>
-          <div class="wc-Form-group wc-Form-group--biggerOffset js-Form-group">
+          <div class="wc-Form-group js-Form-group">
+            {{ form.steps_reproduce.label(class_='wc-Form-label') }} {{ form.steps_reproduce(class_='wc-Form-item wc-Form-item--textarea js-autoExpand', rows=4) }}
+          </div>
+        </div>
+        <div class="r-Grid-cell r-minM--1of2">
+
+          <div class="wc-Form-group wc-Form-group--biggerOffset js-Form-group {% if form.url.errors %}is-error{% endif %}">
             {{ form.browser.label(class_='wc-Form-label') }}
-            <span class="wc-Form-validation">
+            <span class="wc-Form-validation" id="span-browser">
```

Comment:
@cch5ng ah, makes sense. (but I didn't see any other usage of `span-browser` in this PR...)

We have a convention to use `js-thing-for-whatever` classes as general hooks for JS or tests. For example,

https://github.com/webcompat/webcompat.com/blob/master/tests/functional/comments-auth.js#L34
https://github.com/webcompat/webcompat.com/blob/master/tests/functional/comments-auth.js#L51
etc.

Can you create a `js-form-browser-wrapper` (terrible name, pick something that makes sense :p) and use that instead of an ID?